src/templates/forum_conversation/topic_detail.html

```diff
@@ -248,4 +204,12 @@ <h4 class="m-0 subject">
       })
     });
   </script>
+  <style>
+    .vote-container {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      gap: 1rem;
+    }
+  </style>
```

Comment:
We could put these stylings on the `.vote-container` itself to make the code more succinct. In bootstrap, the classes would be:

`d-flex align-items-center gap-1`

Reply:
Considering all of the other stylings are inline -- for uniform sake, it is good to maintain homogeneity.